discord/discord.py

```diff
@@ -32,4 +33,28 @@ def get_locations(token):
 
     return jsonify(output)
 
+@discord.route('/discord/region/<token>', methods=['GET', 'POST'])
```

Comment:
are you going to give multiple region's or just one?

Reply:
Multiple at the moment !region acts like a search function, the most likely value is processed client side as the result, and is used in !map !heatmap